RDF.js

```diff
@@ -828,277 +849,364 @@ function importItem(newItem, node) {
 	var ret = {};
 	var itemType = detectType(newItem, node, ret);
 	var isZoteroRDF = false;
-	if (getFirstResults(node, [n.z+"itemType", n.z+"type"], true)) {
+	if (getFirstResults(node, [n.z + "itemType", n.z + "type"], true)) {
 		isZoteroRDF = true;
 	}
 	newItem.itemType = exports.itemType || itemType;
 	var container = ret.container;
 	var containerPeriodical = ret.containerPeriodical;
 	var containerPublicationVolume = ret.containerPublicationVolume;
-	var containerPublicationIssue = ret.containerPublicationIssue;
+	// var containerPublicationIssue = ret.containerPublicationIssue;
```

Comment:
We don't use it currently and I am not sure how to use it w/o having an example.